README.md

```diff
@@ -40,33 +40,33 @@ To try it out go to your favourite HLS video player such as `https://web.player.
 
 | PARAMETER    | DESCRIPTION                                                                               |
 | ------------ | ----------------------------------------------------------------------------------------- |
-| `url`        | Url path to the original HLS stream (REQUIRED)                                            |
+| `url`        | Url path to the original HLS/MPEG-DASH stream (REQUIRED)                                  |
 | `delay`      | Delay the response, in milliseconds, for a specific segment request                       |
 | `statusCode` | Replace the response for a specific segment request with a specified status code response |
 | `timeout`    | Force a timeout for the response of a specific segment request                            |
 
 ## Corruptions
 
-Currently, the Chaos Stream Proxy supports 3 types of corruptions for HLS streams. These corruptions may be used in combination of one another.
+Currently, the Chaos Stream Proxy supports 3 types of corruptions for HLS and MPEG-DASH streams. These corruptions may be used in combination with one another.
 
-### Specifying Corruption Configurations (HLS)
+### Specifying Corruption Configurations
 
 To specify the configurations for a particular corruption, you will need to add a stringified JSON object as a query parameter to the proxied URL.
 Each corruption has a unique configuration JSON object template. Each object can be used to target one specific segment for corruption.
 e.i. `https://chaos-proxy.prod.eyevinn.technology/api/v2/manifests/hls/proxy-master.m3u8?url=<some_url>?some_corruption=[{i:0},{i:1},{i:2}, ... ,{i:N}]`
 
 Across all coruptions, there are 2 ways to target a segment in a playlist for corruption.
 
-1. `i`: The segments list index in any Media Playlist. For a Media Playlist with 12 segments, `i`=12, would target the last segment.
-2. `sq`: The segments Media Sequence Number. For a Media Playlist with 12 segments, and where `#EXT-X-MEDIA-SEQUENCE` is 100, `sq`=112, would target the last segment. When corrupting a live stream it is recommended to target with `sq`.
+1. `i`: The segment's list index in any Media Playlist, with HLS segments starting at 0 and MPEG-DASH segments starting at 1. For a Media Playlist with 12 segments, `i`=11, would target the last segment for HLS and `i`=12, would target the last segment for MPEG-DASH.
+2. `sq`: The segment's Media Sequence Number. For a Media Playlist with 12 segments, and where `#EXT-X-MEDIA-SEQUENCE` is 100, `sq`=111 would target the last segment for HLS, and `sq`=112 would target the last segment for MPEG-DASH. When corrupting a live stream it is recommended to target with `sq`.

```

Comment:
This was my assumption, but after googling it seems `#EXT-X-MEDIA-SEQUENCE` is only a thing for HLS? So for MPEG-DASH `i` should always be used?

Reply:
This should be clarified if it's the case